src/pip/_internal/cmdoptions.py

```diff
@@ -102,6 +102,15 @@ def getname(n):
     help='Give more output. Option is additive, and can be used up to 3 times.'
 )
 
+no_color = partial(
+    Option,
+    '--no-color',
+    dest='no_color',
+    action='store_true',
+    default=False,
+    help="Suppress colored output"
```

Comment:
nit: a final comma will prevent the need to change this line if anything is added